app/page/home/index.tsx

```diff
@@ -19,12 +28,35 @@ export enum InfoNavigation {
 
 export function Home() {
   const hasNoAccount = useAccounts().length === 0
+
+  const { provider } = useContext(ProviderContext)
+  const { createSimpleAccount } = useAction()
+  const network = useNetwork()
+
+  useEffect(() => {
+    const initSimpleAccount = async () => {
+      const hasSimpleAccountFactory = address.isValid(
+        network.accountFactory[AccountType.SimpleAccount]
+      )
+
+      if (hasNoAccount && hasSimpleAccountFactory) {
+        const account = await SimpleAccount.initWithFactory(provider, {
+          ownerPrivateKey: Wallet.createRandom().privateKey,
+          salt: number.random(),
+          factoryAddress: network.accountFactory[AccountType.SimpleAccount]
+        })
+
+        await createSimpleAccount(`Account 1`, account, network.id)
+      }
+    }
+
+    initSimpleAccount()
+  }, [network.id])
+
```

Comment:
nit: It could be better to return first

```tsx
if (hasNoAccount) {
    return
}
```

Reply:
Thank you!
It should be `if (!hasNoAccount)`, because a new account is created if there is no account.